common/lib/xmodule/xmodule/video_module/video_handlers.py

```diff
@@ -383,6 +383,23 @@ def transcript(self, request, dispatch):
 
         return response
 
+    @XBlock.handler
+    def student_view_state(self, request, suffix=''):
+        """
+        Endpoint to get student view data JSON.
```

Comment:
Could you make this a little more descriptive, so it's clear that this can be used by clients to get more information that can be useful for rendering the student view frontend?

Reply:
Thanks, updated.